update the README with a fuller description of the script and pipeline (#58)
alphagov/zendesk-scripts: README.md

sh scripts/merge-agent-and-group-description.sh
```

### Other Scripts

#### De-duplicate support queue
* The script uses the Zendesk API to identify multiple tickets from the same UserID in the 1st line support queue which match a set of parameters. If there are > 1 tickets with a status of not closed or solved, they are moved to a dedicated Zendesk queue.

> * The script is automated via a Concourse pipeline called 'zendesk-deduplicator'. On a successful merge to master, a new docker container is built, tagged and stored. Following the schedule, the latest version of the container is retrieved and the ruby script executed. Secrets are stored using AWS SSM.

Reviewer: ```suggestion
* The script is automated via a Concourse pipeline called `zendesk-deduplicator`. On a successful merge to master, a new docker container is built, tagged and stored. Following the schedule, the latest version of the container is retrieved and the ruby script is executed. Secrets are stored using AWS SSM.
```

> * The script requires sufficient permissions in Zendesk and is configured to use account email and a token as auth.

Reviewer: ```suggestion
* The script requires sufficient permissions in Zendesk and is configured to use an account email address and an API token for authenticating API requests.
```

> To use the script from the CLI, set the environment variables as above, then:

Reviewer: ```suggestion
To use the script from the CLI, set the environment variables as [above](#add-environmental-variables-to-bashrc-or-to-screen-session), then:
```

Can we add the link to allow the user to find the relevant section about adding environment variables quickly?

1. Edit window required (e.g. 24 hours) in this line, e.g.:
```

> 

Reviewer: ```suggestion
```

Can we delete the empty line?

window_start_time = Time.now - 24 * 3600

> 

Reviewer: ```suggestion
```

Can we delete the empty line?
